src/reader.rs

```diff
@@ -6511,10 +6511,22 @@ impl<'a> Reader<'a> {
             prefix.push_utfstr(&tok);
             prefix.push_utfstr(&common_prefix);
         } else {
-            // Append just the end of the string.
+            // Collapse parent directories and append end of string
```

Comment:
should probably add to the changelog, maybe something like

	- When tab completion results are truncated, the common directory name is omitted.